elyra/pipeline/handlers.py

```diff
@@ -97,3 +99,27 @@ def post(self, *args, **kwargs):
         self.set_header("Content-Type", 'application/json')
         self.finish(json_msg)
         self.flush()
+
+
+class PipelineConfigHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to retrieve pipelines editor configuration"""
+
+    valid_resources = ["palette", "properties"]
+
+    @web.authenticated
+    @gen.coroutine
+    def get(self, resource):
+        if(resource in self.valid_resources):
```

Comment:
I think it might be a good idea to ["unescape" any parameters](https://github.com/elyra-ai/elyra/blob/master/elyra/metadata/handlers.py#L101-L102) before their use.